gruntfile.js

```diff
@@ -175,7 +175,7 @@ module.exports = function (grunt) {
             },
             snippetvNext: {
                 files: {
-                    'AISKU/dist-history/snippet.min.js': ['AISKU/snippet/snippet.js']
+                    'AISKU/snippet/snippet.min.js': ['AISKU/snippet/snippet.js']
```

Comment:
Should delete the dist-history snippet minified file. Also consider deleting the rest of the old sdk version in there as well

Reply:
We want to keep history of shipped versions checked in as well in case someone accidentally deletes from storage account. I'll do another PR to update dist-hist with the bits that we release on Monday.